airflow/contrib/operators/kubernetes_operator.py

```diff
@@ -244,17 +244,35 @@ def poll_job_completion(self, job_name, dependent_containers={'cloudsql-proxy'})
                 time.sleep(self.sleep_seconds_between_polling)
 
                 pod_output = self.get_pods(job_name)
-                job_description = KubernetesJobOperator.retryable_check_output(['kubectl', 'describe', 'job', job_name])
-                matched = re.search(r'(\d+) Running / \d+ Succeeded / (\d+) Failed', job_description)
-                logging.info('Current status is: %s' % matched.group(0))
 
-                running_job_count = int(matched.group(1))
-                failed_job_count = int(matched.group(2))
-                if failed_job_count != 0:
+                job_description = json.loads(KubernetesJobOperator.retryable_check_output(
```

Comment:
please replace the `import json` at the top with`import ujson as json`

Reply:
I changed that import, and re-tested in QA.